### DISCUSSION TITLE
Blank screen on load

### DISCUSSION THREAD

**Discussion by LyndonL37** [VueTorrent/VueTorrent#1036] [Q&A] [answered]
Hi,
I'm just setting up a new instance on qBittorrent and having trouble getting the WebUI to display. I remember having this issue last time I set it up and can't remember what I did to fix it. 
The screen loads with just the blank background in the dark colour, with "qBittorrent" in the title, however no body. 

View page source shows the following:
```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
    <meta name="description" content="VueTorrent" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
    <meta name="apple-mobile-web-app-capable" content="yes" />
    <meta name="mobile-web-app-capable" content="yes" />
    <link rel="icon" href="./favicon.ico" sizes="any">
    <link rel="icon" href="./icon.svg" type="image/svg+xml">
    <link rel="apple-touch-icon" href="./apple-touch-icon.png">
    <meta name="theme-color" content="#000" />
    <title>qBittorrent</title>
    <script type="module" crossorigin src="./assets/index-9b3b9c7e.js"></script>
    <link rel="modulepreload" crossorigin href="./assets/vue-9ef19677.js">
    <link rel="stylesheet" href="./assets/index-adff5842.css">
  <link rel="manifest" href="./manifest.webmanifest" crossorigin="use-credentials"><script id="vite-plugin-pwa:register-sw" src="./registerSW.js"></script></head>
  <body>
    <noscript>
      <strong>We're sorry but Vuetorrent doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
    </noscript>
    <div id="app"></div>
    
  </body>
</html>
```

And the browser console is showing me:
`Failed to load resource: net::ERR_CONNECTION_TIMED_OUT                  index-9b3b9c7e.js:1`

The closest named files in my vuetorrent\assets folder are: index-68d0286b.js and index-30187fe0.js.

Does qBt rename the suffix on the fly for cross site hijacking protection or something? There seems to be a disconnect.

My versions are: 
qBittorrent 4.5.4
VueTorrent 1.7.0

WebUI settings are:
![webui](https://github.com/WDaan/VueTorrent/assets/141901074/9f9d8c9d-0258-489f-9b3a-855557042e17)
I have tried disabling Use HTTPS and all of the Security options without any success.

Thanks for any help.

**Comment by Larsluph** [comment]
The filename is auto-generated at each build, so the name changing is normal. But the `index.html` not linking to the correct name is really weird, maybe multiple VueTorrent instances are overlaping, did you make a clean install when updating the WebUI ?

You have nothing loading because what contains everything of the webui is the `<script type="module" crossorigin src="./assets/index-9b3b9c7e.js"></script>` tag, which refers to the file having a timeout.

**Reply by LyndonL37** [reply]
Hi, thanks for the response.
I have installed both qBt and VT fresh, however I did copy across the qBt settings files from another system. Might that be the culprit?

**Reply by Larsluph** [reply, marked as answer]
If you stayed on the same OS that shouldn't be a problem as long as the paths you specified matches your new environment.

I'd say that your qBittorrent isn't faulty because it serves the main content of the app, however the timeout may indicate a bad config of the web server.

Are you using some kind of reverse proxy or something that may interfere with security protocols ?
If you say that the main js script doesn't exists, can you try renaming the path in the `index.html` file to the correct script on your filesystem ? (the correct one is the largest that is named `index-xxxxxxxx.js`

**Reply by LyndonL37** [reply]
Hi again,
Thanks for the suggestions. No reverse-proxy, however interestingly when I changed the port it was still loading on the old port. I then uninstalled everything and reinstalled it and it was still attempting to connect on the old port and actually serving. 
I think it might have been cached in my router.
Anyway, it's working now.
Thanks a lot for your help!
